app/driver.go

```diff
@@ -37,10 +37,12 @@ const (
 	FlagCSIProvisionerImage     = "csi-provisioner-image"
 	FlagCSIDriverRegistrarImage = "csi-driver-registrar-image"
 	FlagCSIProvisionerName      = "csi-provisioner-name"
+	FlagCSIPodCheckImage        = "csi-pod-check-image"
```

Comment:
Just reuse the manager image. It's already on the node and needs to be run anyway.